roles/postgresql/tasks/postgresql_yum7.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: yum | Initialize postgresql
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: installed
+    update_cache: yes
+  with_items:
+    - postgresql-server
+    - postgresql-contrib
+    - postgresql-devel
+  tags: [db, postgresql]
+
+- name: yum | Initialize Postgresql
+  become: yes
+  command: postgresql-setup initdb
```

Comment:
This command needs idempotence, maybe `creates` some lock file, don't know what else we can do.